<launch>
<!-- Launch file for Localization -->

	<arg name="rviz" default="true" />

    <rosparam file="$(find fast_lio_sam)/config/map_stitch.yaml" command="load" />

	<param name="globalMapVisualizationSearchRadius" type="double" value="1000" />
	<param name="globalMapVisualizationPoseDensity" type="double" value="3" />
	<param name="globalMapVisualizationLeafSize" type="double" value="0.2" />

    <node pkg="fast_lio_sam" type="map_stitch" name="system" output="screen" /> 

	<group if="$(arg rviz)">
		<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find fast_lio_sam)/rviz_cfg/mapping.rviz" />
	</group>

</launch>